src/routes/docs/products/auth/accounts/+page.markdoc

```diff
@@ -1,175 +1,27 @@
-
-final account = Account(client);
-
-final user = await account.updatePrefs(
-    prefs: {
-        "darkTheme": true,
-        "language": "en",
-    }
-);
-```
-```client-apple
-import Appwrite
-
-let client = Client()
-    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1") // Your API Endpoint
-    .setProject("<PROJECT_ID>")                  // Your project ID
-
-let account = Account(client)
-
-let user = try await account.updatePrefs(
-    prefs: [
-        "darkTheme": true,
-        "language": "en"
-    ]
-)
-```
-```client-android-kotlin
-import io.appwrite.Client
-import io.appwrite.services.Account
-
-val client = Client()
-    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1") // Your API Endpoint
-    .setProject("<PROJECT_ID>")                  // Your project ID
-
-val account = Account(client)
-
-val user = account.updatePrefs(
-    prefs = mapOf(
-        "darkTheme" to true,
-        "language" to "en"
-    )
-)
-```
-```graphql
-mutation {
-    accountUpdatePrefs(
-        prefs: "{\"darkTheme\": true, \"language\": \"en\"}"
-    ) {
-        _id
-        name
-        prefs {
-            data
-        }
-    }
-}
-```
-{% /multicode %}
-
-After a user's preferences are updated, they can be retrieved using the [get account preferences](/docs/references/cloud/client-web/account#getPrefs) endpoint.
-
-{% multicode %}
-```client-web
-import { Client, Account } from "appwrite";
-
-const client = new Client()
-    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('<PROJECT_ID>');                 // Your project ID
-
-const account = new Account(client);
-
-const promise = account.getPrefs();
-
-promise.then(function (response) {
-    console.log(response); // Success
-}, function (error) {
-    console.log(error); // Failure
-});
-```
-```client-flutter
-import 'package:appwrite/appwrite.dart';
-
-final client = Client()
-    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('<PROJECT_ID>');                 // Your project ID
-
-final account = Account(client);
-
-final prefs = await account.getPrefs();
-```
-```client-android-kotlin
-import io.appwrite.Client
-import io.appwrite.services.Account
-
-val client = Client(context)
-    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1") // Your API Endpoint
-    .setProject("<PROJECT_ID>")                  // Your project ID
-
-val account = Account(client)
-
-val prefs = account.getPrefs()
-```
-```client-apple
-import Appwrite
-
-let client = Client()
-    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1") // Your API Endpoint
-    .setProject("<PROJECT_ID>")                  // Your project ID
-
-let account = Account(client)
-
-let prefs = try await account.getPrefs()
-```
-```graphql
-query {
-    accountGetPrefs {
-        data
-    }
-}
-```
-{% /multicode %}
+{% arrow_link href="/docs/products/auth/user-preferences" %}
+Learn more about user preferences
+{% /arrow_link %}
```

Comment:
This link seems very random here:

<img width="824" height="510" alt="image" src="https://github.com/user-attachments/assets/151c3217-96e4-46cd-9329-5fceed686821" />

Reply:
Ture, I've removed it

Reply:
True, I've removed it